### PR TITLE
add venv python to path

### PR DESCRIPTION
**Pull request by StanHatko**
@blairdrummond  Please merge this, now has venv Anaconda Python in the path.

### CODE REVIEW

File: crops-pipeline/Dockerfile [hunk 1]
```diff
@@ -55,7 +55,8 @@ RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
 #Start container as non-root user.
 USER $USER
 
-#Activate the virtual environment in the image.
+#Activate the virtual environment in the image and adjust the path.
 SHELL ["/bin/bash", "-c"]
-RUN source venv/bin/activate
+RUN source venv/bin/activate && \
+    export PATH=$USER_HOME/src/venv/bin:$$PATH
```
Comment (blairdrummond) [inline crops-pipeline/Dockerfile:61]: Every `RUN` command runs in a subshell, so I think export will not work. I think you want

https://stackoverflow.com/a/38742545

the PATH variable will not make it to your actual code

Comment (blairdrummond) [reply crops-pipeline/Dockerfile:61]: Same deal with the `source` . You need to add that to the entrypoint I think

Comment (StanHatko) [reply crops-pipeline/Dockerfile:61]: If I specify a custom command to the entry point, will the entrypoint still run? If not, then I think I need to run source in the kfp.dsl.ContainerOp and not in the docker file?

Comment (blairdrummond) [reply crops-pipeline/Dockerfile:61]: Maybe putting it in the ContainerOp is the most explicit route, that would make sense to me

Comment (StanHatko) [reply crops-pipeline/Dockerfile:61]: How does ContainerOp interact with shell? I just noticed that SHELL ["/bin/bash", "-c"] already has the "-c" parameter, maybe that had an effect with the earlier code? Anyway I already have working code so I'll simply move the source and export path to the ContainerOp.

**Comment by StanHatko** [comment]
@blairdrummond  Added new version, these are moved to ContainerOp now.

**Comment by StanHatko** [comment]
There are changes, I removed the source command.

**Review by blairdrummond — APPROVED** [review]
LGTM!

**Pull request merged**